qemu/tests/cfg/blockdev_commit_fio.cfg

```diff
@@ -16,14 +16,14 @@
     rebase_mode = unsafe
     qemu_force_use_drive_expression = no
     mount_point = "/var/tmp"
-    sleep_min = 300
-    sleep_max = 480
-    fio_timeout = 2400
-    commit_job_timeout = 1800
+    sleep_min = 50
+    sleep_max = 100
+    fio_timeout = 300
+    commit_job_timeout = 420
     ppc64le, s390x:
-        commit_job_timeout = 2400
+        commit_job_timeout = 420
```

Comment:
Just,  `commit_job_timeout` is same for all now.   Are you waiting for the  test result from `ppc64le` to decide it ? 
If plan to use same value for all finally,  remove duplicate setting is better.

Reply:
hi @ngu-niny , is this(420s) OK for non-x86 platform testing?

Reply:
420s works for me. @MiriamDeng Could you help to verify the value works on ppc64le?

Reply:
Checking it now, will update it today.